proto/minder/v1/minder.proto

```diff
@@ -732,7 +732,7 @@ message CreateUserResponse {
 }
 
 message DeleteUserRequest {
-    Context context = 1;
+    reserved 1; // deprecated context
```

Comment:
Although we previously prefixed a field with `retired_` when we deprecated it, I think this approach may be cleaner (where we reserve the field ID to avoid reusing it)

Reply:
Works for me!